project/plugins.sbt

```diff
@@ -1,3 +1,10 @@
-addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.13")
-
+ThisBuild / libraryDependencySchemes ++= Seq(
+  "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
+)
+addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.2.0")
```

Comment:
Remove the scalajs plugin
We probably don't need the scoverage plugin either if we're getting code coverage from the new plugins